README.md

```diff
@@ -109,6 +109,24 @@ The text to be included in the PR comment.
 
 **Required**: true
 
+##### Templates
+
+Comment snippet bodies (as well as `comment.header` and `comment.footer`) are [Mustache][https://mustache.github.io/mustache.5.html] templates.
+
+Variables available in the template are:
+- `pr` - the whole response of a [`GET /repos/{owner}/{repo}/pulls/{pull_number}` request](https://docs.github.com/en/rest/reference/pulls#get-a-pull-request).
```

Comment:
Is there any chance that the response contains any secrets that should be filtered out? I don't think so 🤔 I'm inspecting it with the `gh` CLI tool, e.g.
```
gh api /repos/exercism/elixir/pulls/800
```